pkg/kubelet/config/config.go

```diff
@@ -519,6 +519,7 @@ func copyPods(sourcePods []*v1.Pod) []*v1.Pod {
 		pod, err := api.Scheme.Copy(source)
 		if err != nil {
 			glog.Errorf("unable to copy pod: %v", err)
+			continue
```

Comment:
`continue` should be fine. We shouldn't block everything if one pod copying fails -- but when will the deep copy fail?